crates/tests-e2e/src/app.rs

```diff
@@ -90,7 +91,12 @@ pub fn sync_dlc_channels() {
 ///
 /// To call this make sure that you are either outside of a runtime or in a multi-threaded runtime
 /// (i.e. use `flavor = "multi_thread"` in a `tokio::test`).
-pub fn force_close_dlc_channel() {
+pub async fn force_close_dlc_channel(bitcoind: &Bitcoind) {
+    // We need to move the blockchain time forward to be able to publish the buffer transaction
```

Comment:
Some time needs to pass before the buffer transaction can be mined, since there is a timestamp timelock on it. It's probably sufficient to sleep so that the regtest mempool doesn't reject the transaction, but I decided to mine a few blocks too because I believe the blockchain time will only move forward when new blocks are mined (cc @bonomat).